components/FormComponents/SelectInput.jsx

```diff
@@ -7,7 +7,8 @@ const SelectInput = (props) => {
         var selectedOption = props.options ? props.options[0] : {}
         if(props.options) {
             props.options.forEach((option) => {
-                if(option.label === selectedValue || option.value == selectedValue || (option.value).toString() === selectedValue) {
+                if(option.label === selectedValue || option.value == selectedValue || 
```

Comment:
Do we want `option.value == selectedValue` to be `option.value === selectedValue`?

Reply:
i think it breaks if you change it idk why